coordinator/history/history.go

```diff
@@ -170,6 +170,23 @@ func (h *History) WatchLatestTransitions(ctx context.Context) (<-chan LatestTran
 	return transitionCh, nil
 }
 
+// WalkTransitions executes a function for the referenced transition and all its ancestors.
+//
+// The all-zero transition is the root node of all transition trees and is not passed to the closure.
+func (h *History) WalkTransitions(transitionRef [HashSize]byte, consume func([HashSize]byte, *Transition) error) error {
```

Comment:
The rest of the package refers to `transitionRef` as `hash`, so it should be called `hash`/`transitionHash` here, too.